src/config/dynamic-theme-fixes.config

```
@@ -6362,6 +6362,7 @@ a[href*="about/products"]
 .activity-icon
 img[src$="menu_black_24dp.png"]
 #assistive-chips .e2moi img
+.xoLGzf-BIqFsb-haAclf
```

I think we can change it a bit to make it future-proof
```suggestion
button[guidedhelpid="searchbutton"]
```